docs/release-notes/saros-e_15.0.0.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Saros Release Notes
+---
+
+The Saros Team is happy to announce Saros for Eclipse 15.0.0 and Saros for IntelliJ 0.1.0.
+
+Since the last release of Saros for Eclipse we mostly restructured, refactored and reworked the project in more than 1000 commits. From now on we want to use a lot shorter release cycles in order to
+provide fixes and features faster.
+
+Saros for IntelliJ 0.1.0 is the first alpha release version of our IntelliJ version. Please, see the [Saros for IntelliJ release notes](saros-i_0.1.0.md) before you use the new plug-in.
```

Comment:
It seems weird to mention Saros/I in the Saros/E release notes. I think this remark should be removed.

Reply:
Also, the link does not work.
```suggestion
Saros for IntelliJ 0.1.0 is the first alpha release version of our IntelliJ version. Please, see the [Saros for IntelliJ release notes](saros-i_0.1.0.html) before you use the new plug-in.
```